ui/packages/shared/profile/src/MatchersInput/index.tsx

```diff
@@ -115,12 +116,18 @@ export const useLabelValues = (
         profileType !== '' &&
         labelName !== undefined &&
         labelName !== '',
-      staleTime: 1000 * 60 * 5, // 5 minutes
+      staleTime: 1000 * 60 * 2, // 2 minutes
```

Comment:
Why do we need staleness at all? Shouldn’t the timeframe and a success already be sufficient?

note the important thing is that we fetch label names and values based on the selected time (even before hitting search)